deploy.yml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: synapse-deployment
+  namespace: synapse
+  labels:
+    app: synapse
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: synapse
+  template:
+    metadata:
+      labels:
+        app: synapse
+    spec:
+      containers:
+      - name: synapse-container
+        image: !!!IMAGE_LOCATION!!!
+        ports:
+        - containerPort: 8888
+        imagePullPolicy: Always
+        env: 
+        - name: SPRING_PROFILES_ACTIVE
+          value: prdtest
```

Comment:
Noticed this is defaulted to prdtest rather than prd. If this is what you wanted, feel free to ignore this comment!